portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -46,9 +47,14 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
     for (Entity entity : results.asIterable()) {
+      /* Convert the comment's timestamp (ms since Epoch) to a date (d/m/y) */
+      SimpleDateFormat sdf = new java.text.SimpleDateFormat("MMMM d, yyyy");
```

Comment:
I would suggest split the logic as follows

1. What is being stored and returned by the backend?
I think this can be as simple as the epoch. The server logic can be reverted back to just return this.

2. What is being displayed?
On the client side, use the epoch and convert it to local timezone. The client side has this information and this way it displays the right time for each client.

Converting the time to EDT irrespective of the client is definitely wrong on the server side. 

The other way of doing this, the client side figures out the local timezone and sends this information to the server side to perform the conversion. This does make the GET more complicated though.

Reply:
Ah, this is a good idea. I will switch to using just the epoch in Java and having the Javascript do the conversion and formatting.